server/module/services/service/inference_service.py

```diff
@@ -414,48 +414,42 @@ async def run_tts_triton_inference(
         if format == "pcm":
             format = "s16le"
 
-        results = []
-
-        for input in request_body.input:
-            input_string = input.source.replace("।", ".").strip()
-
-            if input_string:
-                inputs, outputs = get_tts_io_for_triton(
-                    input_string, ip_gender, ip_language
-                )
+        inputs_ = [input.source.replace("।", ".").strip() for input in request_body.input]
+        inputs, outputs = get_tts_batched_io_for_triton(inputs_, ip_gender.value, ip_language)
+        response = await self.inference_gateway.send_triton_request(
+            url=service.endpoint,
+            model_name="tts",
+            input_list=inputs,
+            output_list=outputs,
+            headers=headers,
+            request_state=request_state,
+            task_type="tts",
+            request_body=request_body,
+        )
+        raw_audios = response.as_numpy("OUTPUT_GENERATED_AUDIO")
 
-                response = await self.inference_gateway.send_triton_request(
-                    url=service.endpoint,
-                    model_name="tts",
-                    input_list=inputs,
-                    output_list=outputs,
-                    headers=headers,
-                    request_state=request_state,
-                    task_type="tts",
-                    request_body=request_body,
+        results = []
+        for raw_audio in raw_audios:
+            raw_audio = raw_audio[raw_audio != -100]
```

Comment:
For any model that is being deployed, like Prof Hema's TTS, do you think we can fix this as a standard? Ignoring -100's which correspond to padding tokens. Or any plan to make it generalized in the future?